config/testgrids/config.yaml

```diff
@@ -95,9 +95,6 @@ dashboards:
 # Start dashboard groups
 #
 
-dashboard_groups:
-- name: sig-multicluster
-
```

Comment:
/assign @ameukam @dims 

If you could please take a look at the changes in this directory.

Reply:
Sorry for the churn here while I was getting CI to pass, should be ready for review now.